makefiles/libnet.mk

```diff
@@ -29,26 +29,29 @@ endif
 
 libnet-package: libnet-stage
 	# libnet.mk Package Structure
-	rm -rf $(BUILD_DIST)/libnet
+	rm -rf $(BUILD_DIST)/libnet{9/,-dev/}
 	mkdir -p $(BUILD_DIST)/libnet{9/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib,-dev/}
 
 	# libnet.mk Prep libnet-dev
-	cp -a $(BUILD_STAGE)/libnet/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) $(BUILD_DIST)/libnet-dev/
-	rm $(BUILD_DIST)/libnet-dev/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/libnet.9.dylib
-
-	# Prep libnet9
+	# find $(BUILD_STAGE)/libnet/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/ ! -name "libnet.9.dylib" | xargs -i cp -r -a {} $(BUILD_DIST)/libnet-dev/
+	cp -a -r $(BUILD_STAGE)/libnet/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin $(BUILD_DIST)/libnet-dev/
+	cp -a -r $(BUILD_STAGE)/libnet/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/include $(BUILD_DIST)/libnet-dev/
+	cp -a -r $(BUILD_STAGE)/libnet/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/!(libnet.9.dylib) $(BUILD_DIST)/libnet-dev/lib/
+	cp -a -r $(BUILD_STAGE)/libnet/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share $(BUILD_DIST)/libnet-dev/
```

Comment:
1. Ditto
2. `cp -a`, not `cp -a -r`
3. Merge copying of bin, include, share into one statement

Reply:
I was told by @Diatrus to have these as separate statements just above in this chain.

Reply:
He told you to copy the individual files/directory rather than remove it.

Doesn't conflict with doing them in one statement

Reply:
`{bin,share,include}`